ddp-workspace/projects/ddp-rgp/src/styles/_activity.scss

```diff
@@ -377,3 +377,44 @@ ddp-activity-dropdown-picklist-question select {
     padding: 0;
   }
 }
+
+/**
+ * Enrollment nested activities
+ */
+.instance {
+  .instance__divider {
+    display: block;
+    border-top: 2px solid #0C457D;
+    border-bottom: none !important;
+    border-color: #0C457D !important;
+    margin: 25px 0 10px 0;
+  }
+
+  .instance__header {
+    font-size: 1.1rem;
+    font-weight: 500;
+    line-height: 1.6rem;
+    margin-top: 20px;
+  }
+}
+
+.instance:first-child .instance__divider {
+  display: none;
+}
+
+.instance:first-child .instance__header {
+  display: none;
+}
+
+.add-btn {
```

Comment:
I got pretty far with styling the button, but I think I need help with the padding/spacing...

Before:
![rgp_bug_button_before](https://user-images.githubusercontent.com/5713833/116942242-3f4f2280-ac3f-11eb-8f73-ece243319e3d.png)

After:
![rgp_bug_button_after](https://user-images.githubusercontent.com/5713833/116942254-437b4000-ac3f-11eb-8fdc-75cd8e5a57e7.png)

Reply:
How about adding this:
```suggestion
.add-btn {
  display: inline-flex;
  align-items: center;
  justify-content: center;
}
```

This should probably fix the issue with text positioning